src/stories/Library/recommender/recommender.scss

```diff
@@ -0,0 +1,82 @@
+.recommender {
+  background-color: $c-global-tertiary-2;
+
+  @include breakpoint-m {
+    padding: $s-6xl;
+  }
+}
+
+.recommender__title {
+  color: $c-text-primary-white;
+  text-align: center;
+  padding: $s-lg;
+
+  @include breakpoint-m {
+    padding-bottom: $s-6xl;
+  }
+}
+
+.recommender__grid {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  grid-template-rows: 50% 50%;
+
+  @include breakpoint-m {
+    grid-template-columns: 33.33% 33.33% 33.33%;
+  }
+}
+
+.recommender-material {
+  border: 1px solid $c-text-secondary-gray;
+  padding: 30px;
+  display: flex;
+  width: 100%;
+  position: relative;
+  height: 350px;
+
+  @include breakpoint-m {
+    &:last-of-type {
+      display: none;
+    }
+  }
+
+  .recommender-material__favourite {
+    position: absolute;
+    top: 10px;
+    right: 10px;
+  }
+
+  .recommender-material__cover-container {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    margin-top: 10px;
+  }
+
+  .recommender-material__meta {
+    position: absolute;
+    left: 10px;
+    bottom: 10px;
+    width: 100%;
+    padding-right: 10px;
+  }
+
+  .recommender-material__meta__title {
+    all: unset;
```

Comment:
To me unsetting everything seemed somewhat harsh. Then I thought about it and figured it was to remove remove any default link styling. With that in mind I guess this is one way to handle this.

One additional thought was that this means that we only intend for the title to be an actual link. That creates a somewhat small hit area - especially on mobile. Then I thought the entire `.recommender-material` might be a link. However that creates problems with the favourite `<button/>` which is not allowed to be placed inside an `<a/>`. We might be able to place the `<button/>` structually outside the `<a/>` but visually on top of the element. That would be a rather large refactoring so without any specifics regarding what should be linkable and what should not so let's just go with the current implementation 😅 .